http-server/src/main/java/com/facebook/airlift/http/server/HttpServerConfig.java

```diff
@@ -693,4 +705,49 @@ public HttpServerConfig setHttp2StreamIdleTimeout(Duration http2StreamIdleTimeou
         this.http2StreamIdleTimeout = http2StreamIdleTimeout;
         return this;
     }
+
+    public boolean isAuthEnabled()
+    {
+        return authEnabled;
```

Comment:
`authorizationEnabled`

Reply:
We don't use abbrevations in variable names in Presto.